src/xpcc/architecture/platform/devices/stm32/stm32f072-c_r_v-8_b.xml

```diff
@@ -51,7 +51,7 @@
       <vector position="30" name="CEC_CAN"/>
       <vector position="31" name="USB"/>
     </driver>
-    <driver type="adc" name="stm32f0"/>
+<!--     <driver type="adc" name="stm32f0"/> -->
```

Comment:
Right now the "device files" are definitely more of a xpcc build description manual than a generic description of the device. Which is why I don't necessarily think that they would be useful to other projects in the current form.